partEZ/database/migrations/2016_01_27_004712_create_poll_options_table.php

```diff
@@ -15,13 +15,15 @@ public function up()
         if (!Schema::hasTable('poll_options'))
         {
             Schema::create('poll_options', function (Blueprint $table) {
-                $table->increments('oid')->unsigned();
+                $table->increments('oid');
                 $table->integer('pid')->unsigned();
                 $table->string('option');
-                $table->primary(['oid', 'pid']);
+                // $table->primary(array('oid', 'pid'));
```

Comment:
I don't think we will have to delete everything in the migrations table. I'm pretty sure this just worked. The way it was working before, we had to do that every time, and also delete the poll_options table (it wasn't getting deleted).